# Generated by Django 3.0.3 on 2020-03-02 04:33

import contest.models
import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('contest', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='activationmodel',
            name='hash',
            field=models.CharField(db_index=True, default=contest.models.generate_random_string, max_length=30),
        ),
        migrations.AlterField(
            model_name='contestant',
            name='email',
            field=models.EmailField(db_index=True, error_messages={'unique': 'A contestant with that email already exists'}, max_length=254, unique=True),
        ),
        migrations.AlterField(
            model_name='contestant',
            name='username',
            field=models.CharField(db_index=True, error_messages={'unique': 'A contestant with that username already exists'}, max_length=100, primary_key=True, serialize=False, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()]),
        ),
        migrations.AlterField(
            model_name='passwordresetmodel',
            name='hash',
            field=models.CharField(db_index=True, default=contest.models.generate_random_string, max_length=30),
        ),
        migrations.AlterField(
            model_name='question',
            name='number',
            field=models.IntegerField(db_index=True),
        ),
    ]
